vocode/streaming/models/telephony.py

```diff
@@ -53,11 +62,25 @@ class DialIntoZoomCall(BaseModel):
     twilio_config: Optional[TwilioConfig] = None
 
 
-class CallConfig(BaseModel):
+class CallConfigType(str, Enum):
+    BASE = "call_config_base"
+    TWILIO = "call_config_twilio"
+    VONAGE = "call_config_vonage"
+
+
+class CallConfig(TypedModel, type=CallConfigType.BASE.value):
     transcriber_config: TranscriberConfig
     agent_config: AgentConfig
     synthesizer_config: SynthesizerConfig
+    from_phone: str
+    to_phone: str
```

Comment:
should these be optional? and should it be called `BaseCallConfig`? seems more in line with what we've done so far

Reply:
they were optional before, but were always set. now they're no longer optional since we send a `PhoneCallConnectedEvent` with them every time a call connects

good call on the `BaseCallConfig` name